vsts/scripts/publishSdksFromDevToProdStorageAccount.sh

```diff
@@ -40,7 +41,13 @@ function copyBlob() {
     local platformName="$1"
     local blobName="$2"
 
-    if blobExistsInProd $platformName $blobName; then
+    if shouldOverwriteSdk || shouldOverwritePlatformSdk $platformName; then
```

Comment:
The environment variables we set it up during the pipeline execution. When trigger a new build, click the 'run pipeline'->'Advanced options'->'Variables' and add the variable name and value there.

Reply:
Just for the sake of pushing the logic further: should we even make Overwriting the default behavior, rather than the option we enable through an environment variable? I imagine we want to be 100% sure when overwriting anything, but how often does it happen?

Reply:
@qianz2 I like this idea! May we document this new parameter in Oryx somewhere, if not enabled by default?

@daniv-msft that's a good point. However, overwriting by default would come at a time cost because the new SDK is ignored if it's already in the storage account:
![image](https://user-images.githubusercontent.com/86327553/159596325-7ab7bf0f-2ef1-44fb-aa10-7f60b07c0173.png)
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5892922&view=logs&j=3a6ca6a0-31a3-5209-a1db-c1537ced6ff0&t=db5bb8a2-5323-51b3-d173-767115f2d388&l=3255

We could run the pipeline once to see how much longer it takes to overwrite. Currently the pipeline runs in about 3-5 mins: 
![image](https://user-images.githubusercontent.com/86327553/159596692-98d2f341-b2f8-4c17-84d4-2daf9cc455f1.png)
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5892922&view=results

Reply:
@daniv-msft I agree that setting it up by default will have time cost for the build. We want the overwrite process happens when we want to modify the existing platform binaries. Other scenarios, most likely we want to skip overwriting.
@william-msft Thanks for reminding, I've updated the docs.